libs/blocks/modal/modal.js

```diff
@@ -152,9 +152,11 @@ export async function getModal(details, custom) {
   const focusVisible = { focusVisible: true };
   const focusablesOnLoad = [...dialog.querySelectorAll(FOCUSABLES)];
   const titleOnLoad = dialog.querySelector('h1, h2, h3, h4, h5');
+  const isVideoFragment = dialog.querySelector('.fragment:first-child')?.dataset?.path?.includes("modals/videos") || false;
```

Comment:
eslint has some complain about the double quotes

Reply:
I have fixed this error now.